editor/src/core/shared/element-template.ts

```diff
@@ -2720,6 +2720,8 @@ export interface SpecialSizeMeasurements {
   elementGridProperties: GridElementProperties
   containerGridPropertiesFromProps: GridContainerProperties
   elementGridPropertiesFromProps: GridElementProperties
+  rowGap: number | null
```

Comment:
shouldn't these be part of GridContainerProperties?

Reply:
Great question and I struggled with it a bit too. I ended up going this route because it felt to me that those are not grid measurements but akin to the other `[...]gap[...]` ones (gridGap, gap). Also, they are "raw values" while the `Grid[...]Properties` are somewhat derived.

(This also would mean the `Grid[...]Properties` could be renamed to `Grid[...]SomethingElse` where `SomethingElse` could be `measurements`, `layout`, …)

Reply:
ah I just searched and apparently this applies to flex too! okay then